website_sale_custom_filter/models/website_sale_custom_filter_value.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Camptocamp
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+from odoo import api, fields, models
+
+
+class WebsiteSaleCustomFilterValue(models.Model):
+    _name = "website.sale.custom.filter.value"
+    _description = "website.sale.custom.filter.value"
+
+    @api.depends("value_filter_id", "value_filter_id.domain")
```

Comment:
```suggestion
    @api.depends("value_filter_id.domain")
```
this will cover both